boards/vendor/custom_plank/example_runner.board.cmake

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_set_sim_runner_ifnset(example)
```

Comment:
maybe this file should follow the `boards/common` placement, as it is not board specific?

Reply:
Sure, can move it up.